### DISCUSSION TITLE
2009/09/11/%E7%BB%88%E4%BA%8E%E5%8D%B0%E5%88%B7%E5%AE%8C%E6%88%90%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#276] [Announcements]
# 2009/09/11/%E7%BB%88%E4%BA%8E%E5%8D%B0%E5%88%B7%E5%AE%8C%E6%88%90%E4%BA%86

刚收到出版社的信，说书印刷好了。下周就可以拿到样书。

https://blog.qizhen.xyz/2009/09/11/%E7%BB%88%E4%BA%8E%E5%8D%B0%E5%88%B7%E5%AE%8C%E6%88%90%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2009年09月11日 16:03 编辑
笔者很欣赏你的幽默

回复
xiaorong
2009年09月11日 14:20 编辑
干脆再改的彻底点，把书名改成“笔者和LabVIEW”算了。

回复
l
2009年09月11日 14:18 编辑
呵呵,想不到是北航的出版社,北航单片机和嵌入式系统出了不少好书.现在又多了一本关于LabVIEW的.

回复
Qizhen
2009年09月11日 13:59 编辑
我在写书的时候，整理并重写了原来blog上的序。不过，其实我的书稿中还是以第一人称来写的，也就是用的“我”。但是编辑在审稿的时候，把书中所有的“我”字都删去，或换成了“笔者”。我不是专业写书的，我想可能这是写书的规范或者规矩吧，要尽量用第三人称。明后天我有时间的时候把书稿中的前言部分粘贴上来，给大家看一下原稿 :)

回复
xiaorong
2009年09月11日 13:51 编辑
前言里怎么这么多“笔者”啊？好别扭啊。原来blog里写的好端端的，为什么要把“我”改掉啊，还有很多无头句，怎么看怎么别扭

回复
